library/general/test/linuxrc_test.rb

```diff
@@ -46,7 +46,8 @@
   "XBusID"           => "0:2:0",
   "XkbRules"         => "xfree86",
   "XkbModel"         => "pc104",
-  "umount_result"    => "0"
+  "umount_result"    => "0",
+  "reboot_timeout"   => "10"
```

Comment:
I expect here also test for that value, that it really return 10 from method ( which is not case now ).

Reply:
and here it means add some meaningful test below so something like:

```
  describe "#reboot_timeout" do
    it "returns integer value if 'reboot_timeout' is found in install.inf" do
      load_install_inf("reboot_timeout" => "15")
      expect(subject.reboot_timeout).to eq(15)
    end

    it "returns nil if 'reboot_timeout' is not found in install.inf" do
      load_install_inf({})
      expect(subject.reboot_timeout).to eq(nil)
    end
  end
```

add it to other test below that describe other methods.

Reply:
thx, done